### ISSUE TITLE
:open autocomplete doesn't work properly

### ISSUE THREAD

**Issue by user471** [tridactyl/tridactyl#1817]
Tridactyl version: 1.16.3pre3059-d853820
Firefox version: Mozilla Firefox 68.0.2
Operating system: win

:open autocomplete doesn't show most frequently visited pages. 
For example when I use firefox addressbar with `github` text it shows
```
github.com
githust.com/issues
github.com/notifications
```
`:open github` doesn't show any of these pages. 
Also I have historyresults = 50 but it only show 10

**Comment by user471** [comment]
Turns out the default value of `bmarkweight` = 100, so if one have more than 10 github bookmarks they will always be shown in `:open github`. I set bmarkweight to 0 but option to remove bmarks from `:open` would be great.

**Comment by bovine3dom** [comment]
> if one have more than 10 github bookmarks they will always be shown in `:open github`

I don't think this is true. It just means that you would have 10 pages which are treated as if you had visited them 100 times. If you visit any other GitHub page 101 times it would show up on top.

How is `set bmarkweight 0` different to removing bookmarks from `:open`, except for showing sites that you had never visited but bookmarked?

**Comment by user471** [comment]
>I don't think this is true. It just means that you would have 10 pages which are treated as if you had visited them 100 times. If you visit any other GitHub page 101 times it would show up on top.

Technically yes, but for example in my history I have less than 100 urls that have more than 100 visits. It could be just me, but I think 100 is a lot.

>How is set bmarkweight 0 different to removing bookmarks from :open, except for showing sites that you had never visited but bookmarked?

I guess behavior would be the same, but without `getBookmarks` it should be faster
https://github.com/tridactyl/tridactyl/blob/7a4f266cef31edffb35f0c6df9d8aeb40b43adef/src/completions/providers.ts#L84
